api/alembic/env.py

```diff
@@ -0,0 +1,86 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+import sys
+import os
+print(os.getcwd())
+sys.path.append(os.getcwd())
+
+from openapi_server.models import database as db
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
```

Comment:
What if there is not a config?

Reply:
or it is not found, maybe we should exception

Reply:
This is specifically the alembic config, this env.py code isn't ever loaded when we're running the app normally, it's only loaded when making migrations using alembic. And if no alembic.ini file exists, it will error out for other reasons